src/app/App.tsx

```diff
@@ -83,6 +85,17 @@ export const App = (): JSX.Element => {
         </Notification>
       </Section>
     );
+  } else if (
+    configErrors === "Vault request failed" ||
+    configErrors === "Vault connection failed"
```

Comment:
It might be better to move these values to a const/enum and add as potential values to the configErrors return type in the redux store instead of using a generic APIError so that the data flow is clearer. 

 ```tsx
    const VAULT_ERRORS = {
      REQUEST_FAILED: "Vault request failed",
      CONNECTION_FAILED: "Vault connection failed"
    }
```